com.stremio.Stremio.json

```diff
@@ -0,0 +1,210 @@
+{
+  "app-id": "com.stremio.Stremio",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "base": "io.qt.qtwebengine.BaseApp",
+  "base-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "/app/opt/stremio/stremio",
+  "separate-locales": true,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--talk-name=org.a11y.Bus",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--own-name=org.kde.*",
+    "--env=TMPDIR=/var/cache/tmp"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/man",
+    "/bin/python"
+  ],
+  "modules": [
+    {
+      "name": "nodejs",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D bin/node /app/bin/node"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://nodejs.org/dist/latest-v15.x/node-v15.8.0-linux-x64.tar.xz",
+          "sha256": "cdb80f0810e5a076eb6079297073147d2564b1c2292bb4436b3f9e9e4fe18759"
+        }
+      ]
+    },
+    {
+      "name": "stremio",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release"
+      ],
+      "build-options": {
+        "env": {
+          "LIBRARY_PATH": "/app/lib:/usr/lib"
+        }
+      },
+      "post-install": [
+        "install -Dm755 server.js /app/opt/stremio/server.js",
+        "install -Dm644 com.stremio.Stremio.desktop /app/share/applications/com.stremio.Stremio.desktop",
+        "install -Dm644 ./images/stremio.svg /app/share/icons/hicolor/scalable/apps/com.stremio.Stremio.svg",
+        "install -Dm644 ./images/stremio_tray_black.svg /app/share/icons/hicolor/scalable/apps/com.stremio.Stremio-symbolic.svg",
+        "install -Dm644 com.stremio.Stremio.metainfo.xml /app/share/metainfo/com.stremio.Stremio.metainfo.xml",
+        "ln -s /app/bin/node /app/opt/stremio/node",
+        "ln -s /app/bin/QtWebEngineProcess /app/opt/stremio/QtWebEngineProcess"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Stremio/stremio-shell.git",
+          "tag": "master"
```

Comment:
It should use specific commit so builds are reproducible.

Reply:
Changed it. In future this will be the release tag but at the moment the current release will not build so I put the latest commit.